plugins/modules/iosxr_banner.py

```diff
@@ -255,7 +255,7 @@ def run(self):
 def main():
     """main entry point for module execution"""
     argument_spec = dict(
-        banner=dict(required=True, choices=["login", "motd"]),
+        banner=dict(required=True, choices=["login", "motd", "exec"]),
```

Comment:
the changes here done, how are you validating those?
also we need to do changes in rm_templates, argspec files as well and not just here as it is just a doc file, actual changes need to be implemented in argspec, rm_templates file